### PR TITLE
feat(RELEASE-1006): update references to `release-service-utils`

### PR DESCRIPTION
**Pull request by seanconroy2021**
This commit addresses [CVE-2024-3727]( https://github.com/advisories/GHSA-6wvf-f2vw-3425) by bumping images.

Change include:
- Bumping quay.io/konflux-ci/release-service-utils
  to the latest available tag.

Signed-off-by: Sean Conroy <sconroy@redhat.com>

### CODE REVIEW

File: pipelines/rhtap-service-push/README.md [hunk 1]
```diff
@@ -21,6 +21,9 @@
 | taskGitUrl | The url to the git repo where the release-service-catalog tasks to be used are stored | Yes | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision | The revision in the taskGitUrl repo to be used | No | - |
 
+## chnages in 3.11.0
```
Comment (johnbieren) [inline pipelines/rhtap-service-push/README.md]: changes not chnages

File: tasks/add-fbc-contribution/README.md [hunk 2]
```diff
@@ -14,6 +14,10 @@ Task to create a internalrequest to add fbc contributions to index images
 | targetIndex    | targetIndex value updated by update-ocp-tag task                                          | No       |                      |
 | resultsDirPath | Path to results directory in the data workspace                                           | No       |                      |
 
+
+## chnages in 3.1.0
```
Comment (johnbieren) [inline tasks/add-fbc-contribution/README.md]: changes not chnages

File: tasks/base64-encode-checksum/README.md [hunk 3]
```diff
@@ -9,6 +9,8 @@ It returns as a result the blob to sign, that is the result of the base64 encode
 | Name                 | Description                                                        | Optional  | Default value                                         |
 |----------------------|--------------------------------------------------------------------|-----------|-------------------------------------------------------|
 | binaries_dir         | Path inside the image where the binaries to extract are stored     | Yes       | "binaries"                                           |
+## Changes in 1.2.0
```
Comment (johnbieren) [inline tasks/base64-encode-checksum/README.md:13]: missing new line before this one

File: tasks/check-data-keys/README.md [hunk 4]
```diff
@@ -14,6 +14,8 @@ Currently, `releaseNotes` is the only supported system.
 |----------|---------------------------------------------------------|----------|---------------|
 | dataPath | Path to the JSON string of the merged data to use       | No       |               |
 | systems  | The systems to check that all data keys are present for | Yes      | []            |
+## Changes in 0.8.0
```
Comment (johnbieren) [inline tasks/check-data-keys/README.md:18]: missing new line before this one

File: tasks/cleanup-workspace/README.md [hunk 5]
```diff
@@ -9,6 +9,8 @@ Tekton task to delete a given directory in a passed workspace and cleanup Intern
 | subdirectory   | The directory to remove within the workspace                                                                     | No       | -             |
 | delay          | Time in seconds to delay execution. Needed to allow other finally tasks to access workspace before being deleted | Yes      | 60            |
 | pipelineRunUid | The uid of the current pipelineRun. It is only available at the pipeline level                                   | Yes      | ""            |
+## Changes in 0.8.0
```
Comment (johnbieren) [inline tasks/cleanup-workspace/README.md:13]: new line before

File: tasks/update-infra-deployments/README.md [hunk 6]
```diff
@@ -19,6 +19,8 @@
 | sharedSecret                   | Secret in the namespace which contains private key for the GitHub App                        | true     | infra-deployments-pr-creator                                                                                                                     |
 | defaultGithubAppID             | ID of Github app used for updating PR                                                        | true     | 305606                                                                                                                                           |
 | defaultGithubAppInstallationID | Installation ID of Github app in the organization                                            | true     | 35269675                                                                                                                                         |
+## Changes in 1.2.0
```
Comment (johnbieren) [inline tasks/update-infra-deployments/README.md:23]: missing new line before

**Review by johnbieren — CHANGES_REQUESTED** [review]

**Comment by seanconroy2021** [comment]
@johnbieren Fixed

**Review by johnbieren — APPROVED** [review]

**Pull request merged**